forta-uniswap-event-py/package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "uniswap-largeswap-event-agent",
+  "version": "0.0.1",
+  "description": "Detects large ETH swaps",
```

Perhaps mention `Uniswap` in the description:
```suggestion
  "description": "Detects large ETH swaps on the Uniswap V2 exchange",
```